cv_next/app/feed/components/CVItemRSC.tsx

```diff
@@ -1,61 +1,117 @@
 "use client";
+import { getBlurredCv } from "@/app/actions/cvs/getBlurCv";
 import Image from "next/image";
 import Link from "next/link";
-import generateImageUrl from "@/helpers/imageURLHelper";
 import Categories from "@/types/models/categories";
-import { useEffect, useState, cache } from "react";
-import { getBlurredCv } from "@/app/actions/cvs/getBlurCv";
+import { useState, useEffect, cache } from "react";
```

Comment:
Should probably rename this file/merge with `CVItem` if we don't RSC anymore here.